sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSource.scala

```diff
@@ -35,6 +36,7 @@ import org.apache.spark.util.collection.OpenHashSet
  *
  * TODO Clean up the metadata files periodically
  */
+@ThreadSafe
```

Comment:
It's just a tag saying the class is thread safe. I just realized we don't use it in Spark. I will remove it and add document instead.